### PR TITLE
[quant][fx] Move embedding ops to backend_config_dict

### PR DESCRIPTION
**Pull request by jerryzh168**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #75469
* #75389
* #75388
* #75387
* #75386
* #75378
* #75377
* #75318
* __->__ #75317
* #75316

Summary:
att, similar to previous PRs, this one moves dynamically quantized rnn ops
to backend_config_dict

we have some temporary configs in backend_config_dict, but it will be removed soon, we want to migrate
everything to backend_config_dict so that we can enable this path for all the code in the code base, starting
from prepare, then to convert. We can start this process after this PR

Test Plan:
python test/test_quantization.py TestQuantizeFx
python test/test_quantization.py TestQuantizeFxOps
python test/test_quantization.py TestFXNumericSuiteCoreAPIs

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D35423789](https://our.internmc.facebook.com/intern/diff/D35423789)

### CODE REVIEW

**Comment by jerryzh168** [comment]
@jerryzh168 has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D35423789).

**Comment by vkuzo** [comment]
> we have some temporary configs in backend_config_dict, but it will be removed soon, we want to migrate
everything to backend_config_dict so that we can enable this path for all the code in the code base, starting
from prepare, then to convert. We can start this process after this PR

thanks for the context, could we document the code with what is temporary?

**Review by vkuzo — APPROVED** [review]

**Comment by jerryzh168** [comment]
> > we have some temporary configs in backend_config_dict, but it will be removed soon, we want to migrate
> > everything to backend_config_dict so that we can enable this path for all the code in the code base, starting
> > from prepare, then to convert. We can start this process after this PR
> 
> thanks for the context, could we document the code with what is temporary?

yeah we have that in the code

File: torch/ao/quantization/fx/backend_config/quantize_handler.py [hunk 1]
```diff
@@ -57,5 +59,9 @@ def get_activation_ctr(
                     return self.overwrite_output_observer
             return qconfig.activation
 
+        # This is temporary, and will be removed soon
```
Comment (jerryzh168) [inline torch/ao/quantization/fx/backend_config/quantize_handler.py:62]: this function, and also a key in backend_config_dict

**Comment by jerryzh168** [comment]
@jerryzh168 has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D35423789).

**Comment by jerryzh168** [comment]
@jerryzh168 has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D35423789).

**Pull request closed without merging**